h2o-core/src/main/java/water/api/FeatureFlagHandler.java

```diff
@@ -0,0 +1,19 @@
+package water.api;
+
+import water.H2O;
+import water.MRTask;
+import water.api.schemas3.FeatureFlagV3;
+import water.rapids.Rapids;
+import water.rapids.Val;
+
+
+public class FeatureFlagHandler extends Handler {
```

Comment:
unnecessary: there is already an Rapids API that can modify system properties

Reply:
I would agree, the function on the client can evaluate the rapid expression, it doesn't have to be delegated to the backend.

Reply:
@michalkurka I am not sure what you mean, what I had in mind is

`h2o.rapids("(setproperty \"{}\" \"{}\")".format("sys.ai.h2o.name", "value"))`

Reply:
Yes

Reply:
@michalkurka , @honzasterba 
do we really want to completely remove the client function that @koniecsveta previously created (it's been removed after your comments)?
1. No one knows how to use this `rapids` function.
2. No one knows the system properties that we allow them to set.
3. the `set_feature_flag` was self documented.
4. that function was also limited to the flags we want users to be aware of, preventing typos (if user makes a typo in a system property with rapids, he can't notice, it will just create the sys property), or to change a property that is not supposed to be changed...
5. Feature flags are also a very good strategy to rollout experimental features, so this client could be profitable in the future.

Reply:
I dont think modifying feature flags at runtime is something we want to expose/support, I dont think most of the code is designed to work with them changing at runtime, thats why I would stick to the Rapids API for testing.

Reply:
>  I dont think modifying feature flags at runtime is something we want to expose/support.

Well, I personally wish the opposite. If we design sth in such a way that it can be enabled/disabled, it should be exposed rather than frustrate the customer wanting to try or disable that feature.

>  I dont think most of the code is designed to work with them changing at runtime

If it can't be changed at runtime, this is not a feature flag. That's why the function was narrowed to a specific list of features. (I know at least 1 more currently: AutoML XGB on multinode...).

Reply:
Personally I'm fine with current state of the code, I was even suggesting @koniecsveta  to implement the flag API in a separate PR. Just want to know if you think it's worth to create a separate PR.